### PR TITLE
Update danish translation (955)

### PR DESCRIPTION
**Pull request by Stisen1**
Focused translations of adventure map object messages

### CODE REVIEW

**Comment by Stisen1** [comment]
This PR is ready to be reviewed.

File: files/lang/dk.po [hunk 1]
```diff
@@ -5418,7 +5418,7 @@ msgid ""
 "Unfortunately, you do not have the wisdom to understand the spell, and you "
 "are unable to learn it."
 msgstr ""
-
+
```
[the remark this answers is not in the document]
Comment (Stisen1) [reply files/lang/dk.po:5421]: Done

File: files/lang/dk.po [hunk 2]
```diff
@@ -5428,7 +5428,7 @@ msgstr ""
 msgid ""
 "\n"
 "Unfortunately, you have no Magic Book to record the spell with."
-msgstr ""
+msgstr ""
```
[the remark this answers is not in the document]
Comment (Stisen1) [reply files/lang/dk.po:5431]: Done

File: files/lang/dk.po [hunk 3]
```diff
@@ -5822,77 +5930,106 @@ msgid ""
 "The lighthouse is now under your control, and all of your ships will now "
 "move further each day."
 msgstr ""
+"Fyret er nu under din kontrol, og alle dine skibe vil nu kunne sejle længere "
+"om dagen."
 
+#, fuzzy
```
[the remark this answers is not in the document]
Comment (Stisen1) [reply files/lang/dk.po:5936]: Donno. I have no idea what "Fuzzy?" is supposed to mean. The program makes it happen for some reason... This one might have been copied from a further developed dk.po file though, that already has been through the analyzer...

File: files/lang/dk.po [hunk 4]
```diff
@@ -5822,77 +5930,106 @@ msgid ""
...
 "wish to enter?"
 msgstr ""
+"Du støder på en forladt guldmine. Minen ser ud til at være hjemsøgt. Vil du "
+"ind?"
 
 msgid "You beat the Ghosts and are able to restore the mine to production."
 msgstr ""
+"Du besejrer spøgelserne og er i stand til at genoprette minens produktion."
 
+#, fuzzy
```
[the remark this answers is not in the document]
Comment (Stisen1) [reply files/lang/dk.po:5951]: Removed

File: files/lang/dk.po [hunk 5]
```diff
@@ -5902,113 +6039,162 @@ msgid ""
...
 
 msgid "Trolls living under the bridge challenge you. Will you fight them?"
-msgstr ""
+msgstr "Trolde, der bor under broen, udfordrer dig. Vil du bekæmpe dem?"
 
 msgid ""
 "A few Trolls remain, cowering under the bridge. They approach you and offer "
 "to join your forces as mercenaries. Do you want to buy any Trolls?"
 msgstr ""
+"Der er nogle få trolde tilbage. som kryber sammen under broen. De henvender "
```
[the remark this answers is not in the document]
Comment (Stisen1) [reply files/lang/dk.po:6135]: Done

File: files/lang/dk.po [hunk 6]
```diff
@@ -5822,77 +5930,106 @@ msgid ""
...
 "position themselves around it. In unison they say: \"Mother Earth would like "
 "to offer you a few of her troops. Do you want to recruit Earth Elementals?\""
 msgstr ""
+"Idet du nærmer dig det boblende mudderhul, begynder væsner at kravle ud og "
+"placere sig omkring den. I kor siger de: \"Moder Jord vil gerne tilbyde dig "
+"et par af sine tropper. Vil du rekruttere Jord-elementaler?\""
 
 msgid "You enter the structure of white stone pillars, and find nothing."
 msgstr ""
+"Da du går ind i en bygning af hvide stensøjler, men du finder intet af værdi."
```
[the remark this answers is not in the document]
Comment (Stisen1) [reply files/lang/dk.po:6032]: .... Only if you want the sentence to be invalid... "Da" has been removed.

File: files/lang/dk.po [hunk 7]
```diff
@@ -5715,35 +5804,48 @@ msgid ""
...
+"Du støder på et gammelt artefakt. Lige som du rækker ud efter den, springer "
+"en flok slyngler ud af krattet, for at beskytte deres stjålne bytte."
 
 msgid ""
 "Through a clearing you observe an ancient artifact. Unfortunately, it's "
 "guarded by a nearby %{monster}. Do you want to fight the %{monster} for the "
 "artifact?"
 msgstr ""
+"Gennem en lysning observerer du et gammelt artefakt. Desværre er den "
+"bevogtet af en nærliggende %{monster}. Vil du kæmpe mod %{monster}en for at "
```
[the remark this answers is not in the document]
Comment (Stisen1) [reply files/lang/dk.po:5816]: Your right. It has been changed to something else, that now works for all the possible forms.

File: files/lang/dk.po (hunk 3 above)
[the remark this answers is not in the document]
Comment (Stisen1) [reply files/lang/dk.po]: Well that sure is nice to know after spending 50+ hours on translating the game ;-)

File: files/lang/dk.po [hunk 8]
```diff
@@ -5715,35 +5804,48 @@ msgid ""
...
 msgstr ""
+"Diskretion er den bedste del af tapperhed, og du beslutter dig for at undgå "
+"denne kamp for i dag."
 
 msgid ""
 "After spending hours trying to fish the chest out of the sea, you open it "
 "and find %{gold} gold pieces."
 msgstr ""
+"Efter at have brugt timevis på at prøve at fiske kisten op af havet, åbner "
+"du den og finder %{guld} guldstykker."
```
Comment (ihhub) [inline files/lang/dk.po:5834]: Misspelled **%{gold}**. This is not a translatable part of the text.

File: files/lang/dk.po [hunk 9]
```diff
@@ -5758,16 +5860,22 @@ msgid ""
 "After scouring the area, you fall upon a hidden chest, containing the "
 "%{gold} gold pieces."
 msgstr ""
+"Efter at have gennemsøgt området, falder du over en skjult kiste, der "
+"indeholder %{guld} guldstykker."
```
Comment (ihhub) [inline files/lang/dk.po:5864]: Misspelled **%{gold}**. This is not a translatable part of the text.

File: files/lang/dk.po [hunk 10]
```diff
@@ -6030,14 +6217,19 @@ msgid ""
 "All of the %{monsters} you have in your army have been trained by the battle "
 "masters of the fort. Your army now contains %{monsters2}."
 msgstr ""
+"Alle de %{monstre}, du har i din hær, er blevet trænet af fortets "
```
Comment (ihhub) [inline files/lang/dk.po:6220]: Misspelled **%{monsters}**. This is not a translatable part of the text.

**Review by ihhub — CHANGES_REQUESTED** [review]
Hi @Stisen1 , I put several comments in this pull request. Would you mind to take a look at them?

File: files/lang/dk.po (hunk 10 above)
Comment (Stisen1) [reply files/lang/dk.po]: Done

File: files/lang/dk.po (hunk 9 above)
Comment (Stisen1) [reply files/lang/dk.po]: Done

File: files/lang/dk.po (hunk 8 above)
Comment (Stisen1) [reply files/lang/dk.po]: Done

**Review by ihhub — APPROVED** [review]

**Comment by ihhub** [comment]
@Stisen1 , thank you very much for the update!

**Pull request merged**